py/torch_tensorrt/dynamo/utils.py

```diff
@@ -183,9 +183,16 @@ def parse_complex_tensor_structs(
 
         return torchtrt_inputs_dict
 
+    elif isinstance(inputs, (torch.SymBool, torch.SymFloat, torch.SymInt)):
+        raise ValueError(
+            f"Detected Torch symbolic input type {type(inputs)} during input parsing. "
+            "Symbolic inputs are not currently allowed; please specify dynamic=False "
```

Comment:
Should it be dynamic=None ?

Reply:
`dynamic=None` can still allow dynamic shapes in the compilation (but prefers static), but `torch.compile` does not support dynamic shapes at all, so I thought this should be `False` to avoid dynamic shapes altogether